docs/multi-homing.md

```diff
@@ -191,6 +191,47 @@ spec:
     imagePullPolicy: IfNotPresent
     name: agnhost-container
 ```
+
+### Setting static IP addresses on a pod
+The user can specify attachment parameters via
+[network-selection-elements](https://github.com/k8snetworkplumbingwg/network-attachment-definition-client/blob/63033d5c63d1cf56f924a5454c8f2ac444b6736d/pkg/apis/k8s.cni.cncf.io/v1/types.go#L137)
+, namely IP, MAC, and interface name.
+
+Refer to the following yaml for an example on how to request a static IP for a
+pod, a MAC address, and specify the pod interface name.
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  annotations:
+    k8s.v1.cni.cncf.io/networks: '[
+      {
+        "name": "l2-network",
+        "mac": "02:03:04:05:06:07",
+        "interface": "myiface1",
+        "ips": [
+          "192.168.200.20/24"
```

Comment:
nit: use the doc range `192.0.2.0/24` etc

Reply:
Done.